src/main/java/org/tikv/common/TiSession.java

```diff
@@ -166,6 +178,10 @@ private static VersionInfo getVersionInfo() {
   }
 
   private synchronized void warmUp() {
+    if (conf.isTest()) {
+      logger.info("skip warm up in test mode");
```

Comment:
why skip warmup in test mode?

Reply:
1. `PDMockServerTest` is the base class of `MockServerTest`.
2. `TiSession` is initialized in `PDMockServerTest`, and it calls `warmup`.
3. `warmup` calls `getAllStores` and  `rawGet`.

This results in a dependent hell. We should ignore this anti-pattern warmup operation in `TiSession`, otherwise, we could not launch `MockServerTest` propperly.

Reply:
Ignored warmup by `setEnableWarmup(false)`